src/test/scala-2/chiselTests/FixedIOModuleSpec.scala

```diff
@@ -320,4 +320,21 @@ class FixedIOModuleSpec extends AnyFlatSpec with Matchers with FileCheck {
     ChiselStage.emitCHIRRTL(new Bar) should include("module Foo :")
   }
 
+  "FixedIOModule" should "create a module with flattened IO with clock and reset" in {
+    class Foo(width: Int) extends FixedIOModule[UInt](UInt(width.W)) {
+      override def resetType = Module.ResetType.Synchronous
+      io :<>= DontCare
+    }
+
+    ChiselStage
+      .emitCHIRRTL(new Foo(8))
+      .fileCheck()(
+        """
+      CHECK: module Foo :
+      CHECK:   input clock : Clock
+      CHECK:   input reset : UInt<1>
+      CHECK:   output io : UInt<8>
+      """
```

Comment:
```suggestion
        """| CHECK: module Foo :
           | CHECK:   input clock : Clock
           | CHECK:   input reset : UInt<1>
           | CHECK:   output io : UInt<8>
           |""".stripMargin
```
Prefer consistent style using `|`